packages/protocol-utils/src/transformer_utils.ts

```diff
@@ -148,7 +148,7 @@ export function decodePayTakerTransformerData(encoded: string): PayTakerTransfor
 }
 
 /**
- * ABI encoder for `PayTakerTransformer.TransformData`
```

Comment:
Maybe better to just leave this package alone and worry about it in the v4 branch

Reply:
okay I've removed it